css/bootstrap/privatebin.css

```diff
@@ -87,6 +87,10 @@ body.loading {
 	word-wrap: break-word;
 }
 
+#plaintext > a {
```

Comment:
In the interest of not repeating yourself, how about merging this with the rule above on line 82?
```suggestion
#pastelink > a, #plaintext > a {
```

Reply:
```suggestion
#pastelink > a, #plaintext  a,  {
```

I made a mistake…

Reply:
That wasn't quite what I had in mind, but I wanted to get this merged, so I resolved the conflict in the merge commit into master. Also realized this (and some other selectors) could get de-duplicated into the common.css.